src/scripts/modules/media/body/body.coffee

```diff
@@ -152,6 +152,31 @@ define (require) ->
             # NOTE: `.toggleClass()` explicitly requires a `false` (not falsy) 2nd argument
             $el.toggleClass('ui-has-child-title', $title.length > 0)
 
+          # Create subtitles in each chapter with corresponding ids as their href
+          total_subtitle = $temp.find("[data-depth='1']")
+          total_id = total_subtitle.children('[data-type="title"]')
```

Comment:
Ah, I was asked about the diff in names w/ and w/o $ - good to know what they're there for.